no member named 'check' in namespace 'eosio'

I'm finding it practically impossible to get any work done with v1.6.1 and I would dearly love to downgrade but the 1.5.0 version I was previously using is no longer available from brew... so I just keep plodding on...

```bash
$ cat > t.cpp <<EOF
#include <eosio/system.hpp>
EOF
$ eosio-cpp t.cpp
```
begets:

> In file included from /Users/ekkis/dev/eos/t.cpp:1:
> In file included from /usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/../include/eosiolib/contracts/eosio/system.hpp:6:
> /usr/local/Cellar/eosio.cdt/1.6.1/opt/eosio.cdt/bin/../include/eosiolib/contracts/eosio/../../core/eosio/time.hpp:157:20: error: 
>       no member named 'check' in namespace 'eosio'
>             eosio::check( std::numeric_limits<uint32_t>::max() - slot >= 1, "block timestamp o...
>             ~~~~~~~^
> Warning, empty ricardian clause file
> Warning, empty ricardian clause file
> 1 error generated.
> Error while processing /Users/ekkis/dev/eos/t.cpp.
> abigen error

god almighty.  I'm including system.hpp because they bloody removed `now()` (which I used to write timestamps) so now I have to call `current_time_point()`, for which I need to include system.hpp

so this is super simple but it won't compile.  what do I need?

--- a/libraries/eosiolib/core/eosio/time.hpp
+++ b/libraries/eosiolib/core/eosio/time.hpp
@@ -1,6 +1,7 @@
 #pragma once
 #include <stdint.h>
 #include <string>
+#include "check.hpp"
 #include "serialize.hpp"
 
 namespace eosio {

